debian/changelog

```diff
@@ -1,3 +1,9 @@
+intel-compute-runtime (24.35.30872.27-1~24.10~ppa2) oracular; urgency=medium
```

Comment:
@androniychuk why do we have `-1` after the upstream version `24.35.30872.27` ?

Reply:
because at intel we have -<build_number> after the version
so because this wasn't a direct upload from intel's src pkgs, we simply have a -1

Reply:
Got it, is there way you can modify it when you do the merge request to have a consistent version scheme with other packages in graphics PPA ?

Reply:
I don't want to put an "-ubuntu" version on this yet, once we get in static linking we can change the versions scheme
For now this works

Reply:
yes, this version scheme also works, adding `-ubuntu` is not mandatory, just a matter of being consistent, i will approve and merge it for now